docker-compose.yml

```diff
@@ -11,4 +11,22 @@ services:
       - POSTGRES_DB=pipeline
     #   Map port 6432 (localhost) to port 5432 on the container
     ports:
-      - "6432:5432"
\ No newline at end of file
+      - "6432:5432"
+
+  gitlab:
+    image: 'gitlab/gitlab-ce:latest'
+    restart: always
+  #  hostname: 'gitlab-pipeline.cmwl'
```

Comment:
Why do we need this?

Reply:
well, that line was from an official example (https://gitlab.com/gitlab-org/omnibus-gitlab/raw/master/docker/docker-compose.yml),
so I commented out that line and left it just in case

Reply:
I don't think we will need it soon. 
Could you delete it, please?

Reply:
sure, done